external/googleapis/update_libraries.sh

```diff
@@ -49,6 +49,11 @@ declare -A -r LIBRARIES=(
       "@com_google_googleapis//google/cloud/clouddms/logging/v1:logging_proto"
   )"
   ["dialogflow"]="@com_google_googleapis//google/cloud/dialogflow/v2:dialogflow_proto"
+  ["eventarc"]="$(
+    printf ",%s" \
+      "@com_google_googleapis//google/cloud/eventarc/v1:eventarc_cc_grpc" \
```

Comment:
No action needed: normally we use the `*_proto` (vs. `*_cc_grpc`) rules here.  Seems like both work, and `*_cc_grpc` are easier to find.  We may want to fix all these in a separate PR.

Reply:
Ack.